Vic2ToHoI4/Source/HOI4World/HoI4Localisation.cpp

```diff
@@ -945,15 +952,20 @@ std::string HoI4Localisation::getLanguageCode(const std::string& language)
 void HoI4Localisation::insertScriptedLocalisation(
 	const std::string& localisationKey,
 	const std::string& replacementKey,
-	HoI4::ScriptedLocalisation& scriptedLocalisation
+	HoI4::ScriptedLocalisation& scriptedLocalisation,
+	const std::set<std::string>& majorIdeologies
 ) const
 {
-	const std::regex extractRegex("([A-Z]+)_([a-z]+)_ADJ");
+	const std::regex extractRegex("([A-Z0-9]+)_([a-z]+)_ADJ");
```

Comment:
heh, regex like this caused us to have k_leon as a country tag in Vic2. You'd have k_leon_ADJ and be none the wiser. :) 
Put {3} beside tag to be sure.